waspc/cli/src/Wasp/Cli/Command/CreateNewProject.hs

```diff
@@ -45,9 +48,17 @@ createProjectOnDisk
     } = do
     cliSendMessageC $ Msg.Start $ "Creating your project from the \"" ++ show template ++ "\" template..."
     case template of
-      RemoteStarterTemplate metadata ->
-        createProjectOnDiskFromRemoteTemplate absWaspProjectDir projectName appName $ _path metadata
+      GhRepoStarterTemplate ghRepoRef metadata ->
+        createProjectOnDiskFromGhRepoTemplate absWaspProjectDir projectName appName ghRepoRef $ _path metadata
       LocalStarterTemplate metadata ->
         liftIO $ createProjectOnDiskFromLocalTemplate absWaspProjectDir projectName appName $ _path metadata
       AiGeneratedStarterTemplate ->
         AI.createNewProjectInteractiveOnDisk absWaspProjectDir appName
+
+-- | This function assumes that the project dir was created inside the current working directory.
+printGettingStartedInstructionsForProject :: NewProjectDescription -> IO ()
+printGettingStartedInstructionsForProject projectDescription = do
+  let projectDirName = init . SP.toFilePath . SP.basename $ _absWaspProjectDir projectDescription
+  let instructions = getTemplateStartingInstructions projectDirName $ _template projectDescription
+  putStrLn $ Term.applyStyles [Term.Green] $ "Created new Wasp app in ./" ++ projectDirName ++ " directory!"
```

Comment:
![Screenshot 2024-01-09 at 11 29 57](https://github.com/wasp-lang/wasp/assets/2223680/f9d5e3ab-f7bd-4cf5-9d89-9fadaaf66c8b)

I'd recommended introducing new lines, so the output looks like this:

```
Created new Wasp app in ./xoxo directory!

To run your new app, do:
    cd xoxo/app
    wasp db start

Then open new terminal window/tab in that same dir and do:
    wasp db migrate-dev
    cp .env.server.example .env.server
    wasp start
```

Reply:
I turned it into this, for saas, because it is more complex than other examples (due to having to open another terminal window, which makes explaining harder):

![image](https://github.com/wasp-lang/wasp/assets/1536647/4e3afd05-ab5b-497e-b238-674aa592f8ad)

Also added some new lines for other stuff.